TTHAnalysis/python/tools/nanoAOD/fastCombinedObjectRecleaner.py

```diff
@@ -91,13 +91,16 @@ def initReaders(self,tree):
             if coll==self.tauc: _vars.extend(self.vars_taus+self.vars_taus_int+self.vars_taus_uchar)
             if coll==self.jc: _vars.extend(self.vars_jets+self.vars_jets_int+self.vars_jets_nooutput)
             for B in _vars:
-                setattr(self,"%s_%s"%(coll,B), tree.arrayReader("%s_%s"%(coll,B)))
+                if type(B) == tuple:
+                    setattr(self,"%s_%s"%(coll,B[1]), tree.arrayReader("%s_%s"%(coll,B[1])))
```

Comment:
shouldn't the first `B[1]` be a `B[0]` ?
otherwise what's the point of having both?

Reply:
The need for the tuple is in the collectionSkimmer: that way Jet_pt_nom is copied to Jet_Recl_pt and can then be used by other modules downstream. 
But, yes, I guess it's better to have B[0] here and "pt" in L103 (it's just a naming convention)